packages/sanity/src/core/form/inputs/arrays/common/list.tsx

```diff
@@ -61,22 +61,24 @@ function sortingStrategy(axis: Axis) {
 }
 
 function SortableList(props: ListProps) {
-  const {items, axis, onItemMove, children, ...rest} = props
+  const {items, axis, onItemMove, onItemMoveStart, onItemMoveEnd, children, ...rest} = props
 
   const sensors = useSensors(useSensor(PointerSensor), useSensor(KeyboardSensor, SENSOR_OPTIONS))
 
   const handleDragEnd = useCallback(
     (event: DragEndEvent) => {
       const {active, over} = event
 
+      onItemMoveEnd?.()
```

Comment:
Feels like`onItemMoveEnd` should be invoked *after* `onItemMove`, so that the order becomes move-start, move, move-end?

Reply:
Good point 👍